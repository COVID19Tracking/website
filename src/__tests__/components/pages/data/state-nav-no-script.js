import React from 'react'
import renderer from 'react-test-renderer'
import StateNavNoScript from '../../../../components/pages/data/state-nav-no-script'

describe('Components : Pages : Data : Navigation with no JS enabled', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <StateNavNoScript
          stateList={[
            {
              node: { state: 'AK' },
            },
            {
              node: { state: 'CA' },
            },
          ]}
        />,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
