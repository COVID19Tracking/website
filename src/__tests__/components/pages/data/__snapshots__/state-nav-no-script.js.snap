// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Pages : Data : Navigation with no JS enabled renders correctly 1`] = `
<div
  className="js-disabled js-disabled-block state-nav-no-js stateNav"
>
  <h3>
    Jump to a state:
  </h3>
  <ul>
    <li>
      <a
        href="#state-ak"
      >
        AK
      </a>
    </li>
    <li>
      <a
        href="#state-ca"
      >
        CA
      </a>
    </li>
  </ul>
</div>
`;
